sqlx-core/src/sqlite/connection/mod.rs

```diff
@@ -40,6 +43,15 @@ impl SqliteConnection {
     pub fn as_raw_handle(&mut self) -> *mut sqlite3 {
         self.handle.as_ptr()
     }
+
+    pub fn create_collation(
+        &mut self,
+        name: &str,
+        collation: impl Fn(&str, &str) -> Ordering + Send + Sync + UnwindSafe + 'static,
```

Comment:
nit: can we call this `compare` instead of `collation` ? to me the pair of name and "compare" are what I would call a "collation" object